packages/components/src/components/list/_list.scss

```
@@ -20,13 +20,18 @@
 @mixin lists {
   .#{$prefix}--list--nested,
   .#{$prefix}--list--unordered,
-  .#{$prefix}--list--ordered {
+  .#{$prefix}--list--ordered,
+  .#{$prefix}--list--ordered_native {
```

```suggestion
  .#{$prefix}--list--ordered--native {
```

To follow our BEM convention, would it make sense to do `list--ordered--native`?